# value = angle(deg)/0.3

laser_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: laser_roll_joint
    joint_speed: 0.5
    motor:
        id: 2
        init: 204 # 0 deg
        min: 102 # -30 deg
        max: 306 # +30 deg

laser_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: laser_pitch_joint
    joint_speed: 0.5
    motor:
        id: 1
        init: 511 # 0 deg
        min: 409 # -30 deg
        max: 613 # +30 deg

kinect_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: kinect_pitch_joint
    joint_speed: 0.5
    motor:
        id: 6
        init: 511 # 0 deg
        min: 425 # -25.8 deg
        max: 654 # +43.0 deg

kinect_yaw_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: kinect_yaw_joint
    joint_speed: 1.0
    motor:
        id: 5
        init: 511 # 0 deg
        min: 349 # -48.7 deg
        max: 673 # +48.7 deg
